packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -54,20 +52,19 @@ def __init__(
         :param contract_interface: the contract interface.
         """
         super().__init__(contract_id, contract_config, contract_interface)
-        self.is_items_created = False
-        self.is_items_minted = False
-        self.token_ids = []  # type: List[int]
         self.item_ids = []  # type: List[int]
 
     def create_item_ids(self, token_type: int, token_ids: List[int]) -> None:
```

Comment:
I still don't understand why we have the `token_ids` here ? Why can we not simple have a signature like this:
```
def create_token_ids(self, token_type: int, nb_tokens: int) -> None:
```

Reply:
And please remove the reference to `items` throughout this file. we should only use the term `token`